storm-core/src/jvm/org/apache/storm/utils/DisruptorQueue.java

```diff
@@ -63,6 +63,18 @@
     private static final String PREFIX = "disruptor-";
     private static final FlusherPool FLUSHER = new FlusherPool();
 
+    private static int getNumFlusherPoolThreads() {
+        int numThreads = 100;
+        try {
+            String threads = System.getProperty("num_flusher_pool_threads", "100");
```

Comment:
Rethinking this a bit: this will be going to be undocumented feature.
Given that we don't reuse this for outside of Storm project, how about adding property to Config and calling Utils.readStormConfig() to read the value?